builder/dockerfile/dispatchers.go

```diff
@@ -157,6 +158,7 @@ func add(req dispatchRequest) error {
 	if err != nil {
 		return err
 	}
+	copyInstruction.flChown = flChown.Value
```

Comment:
Shouldn't people using `ADD` for remote files be converting things to `RUN curl` + `COPY` anyway? So by not adding it to `ADD` we're helping nudge people in the right direction.  It doesn't really make sense to have two almost identical instructions.